pkg/scalers/couchdb_scaler.go

```diff
@@ -177,8 +175,6 @@ func parseCouchDBMetadata(config *ScalerConfig) (*couchDBMetadata, string, error
 		connStr = "http://" + addr
 	}
 
-	// FIXME: DEPRECATED to be removed in v2.12
-	meta.metricName = GenerateMetricNameWithIndex(config.ScalerIndex, kedautil.NormalizeString(fmt.Sprintf("coucdb-%s", meta.dbName)))
```

Comment:
I think this shouldn't be removed as is generated for internal use. It doesn't use a user provided name

Reply:
Yes. I added it back